trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-reports.component-test.js

```diff
@@ -24,7 +22,14 @@ describe(page, () => {
   });
 
   const getWrapper = async () => {
-    const reportPeriodSummaries = await getReportReconciliationSummariesViewModel(MOCK_UTILISATION_REPORT_RECONCILIATION_SUMMARY, 'user-token');
+    const reportPeriodSummaries = [
```

Comment:
Could we update this function to no longer be `async`? We should then be able to remove all the `await`s below as well :)

Reply:
Ah nice catch!